Jenkinsfile

```diff
@@ -21,6 +21,7 @@ def aarch64verifyTargets = [
   'aarch64-verify-install-ubuntu-xenial',
   'aarch64-verify-install-ubuntu-bionic',
   'aarch64-verify-install-centos-7',
+  'aarch64-verify-install-fedora-29',
```

Comment:
Do we reintroduce the fedora 29?

Reply:
No unfortunately I think that was the side-effect of adding this on top of PR #95, going to re-apply that particular patch